commands/FBPrintCommands.py

```diff
@@ -371,3 +372,28 @@ def run(self, arguments, option):
 
     print_command = 'po (NSString *)[[NSString alloc] initWithData:{} encoding:{}]'.format(arguments[0], enc)
     lldb.debugger.HandleCommand(print_command)
+
+class FBPrintTargetActions(fb.FBCommand):
+
+  def name(self):
+    return 'pactions'
+
+  def description(self):
+    return 'Print the actions and targets of a control.'
+
+  def args(self):
+    return [ fb.FBCommandArgument(arg='control', type='UIControl *', help='The control to inspect the actions of.') ]
+
+  def run(self, arguments, options):
+    control = arguments[0]
+    targets = fb.evaluateObjectExpression('[[{control} allTargets] allObjects]'.format(control=control))
+    targetCount = fb.evaluateIntegerExpression('[{targets} count]'.format(targets=targets))
+
+    for index in range(0, targetCount):
+      target = fb.evaluateObjectExpression('[{targets} objectAtIndex:{index}]'.format(targets=targets, index=index))
+      actions = fb.evaluateObjectExpression('[{control} actionsForTarget:{target} forControlEvent:0]'.format(control=control, target=target))
+
+      targetDescription = fb.evaluateExpressionValue('(id){target}'.format(target=target)).GetObjectDescription()
+      actionsDescription = fb.evaluateExpressionValue('(id)[{actions} componentsJoinedByString:@","]'.format(actions=actions)).GetObjectDescription()
```

Comment:
Ah yes, I meant to include an example output, thanks for asking!

For a view controller, something like:

`<XXXViewController: 0xdddddddd>: didTapButton:`

The output would be messier if the target provides a verbose `-description` method. Perhaps this shouldn't rely on the built in `-description` and should instead manually format the `<Class: 0xdddddddd>` string.